code/modules/mob/living/carbon/human/human.dm

```diff
@@ -194,14 +194,18 @@
 		//it might be 'bubblewrapping' given that this rhymes with 'hugboxing'
 		if((tmob.a_intent == I_HELP || tmob.restrained()) && (a_intent == I_HELP || restrained()))
 			if((canmove && tmob.canmove) && (!tmob.buckled && !tmob.buckled_mob))
-				var/turf/oldloc = loc
-				loc = tmob.loc
-				tmob.loc = oldloc
-				now_pushing = 0
-				for(var/mob/living/carbon/slime/slime in view(1,tmob))
-					if(slime.Victim == tmob)
-						slime.UpdateFeed()
-				return
+			
+				if(locate(/obj/structure/table/, loc) && locate(/obj/structure/table/, tmob.loc) || !locate(/obj/structure/table/, loc) && !locate(/obj/structure/table/, tmob.loc))

```

Comment:
Are you sure there isn't some proc you can call to check if the bumped mob can move on the tile?